src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.ts

```diff
@@ -162,14 +158,6 @@ export class DotEditPageToolbarComponent implements OnInit, OnChanges {
         }
     }
 
-    private getWorkflowOptions(): MenuItem[] {
-        return this.pageWorkflows.map((workflow: Workflow) => {
-            return {
-                label: workflow.name
-            };
-        });
-    }
-
     private setLockerState() {
         if (!this.lockerModel && this.mode === PageMode.EDIT) {
             this.mode = PageMode.PREVIEW;
```

Comment:
do remove all this not change anything into the test?

Reply:
No, I wasn't testing that because I knew it was going away.